ci/container/Dockerfile

```diff
@@ -55,6 +55,25 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
     cd .. && \
     rm -rf e2fsdroid
 
+# Install an older libtinfo for rules_haskell
+RUN mkdir libtinfo && \
+    cd libtinfo && \
+    wget "http://security.ubuntu.com/ubuntu/pool/universe/n/ncurses/libtinfo5_6.4-2ubuntu0.1_amd64.deb" && \
+    apt -yqq install ./libtinfo5_6.4-2ubuntu0.1_amd64.deb && \
+    cd .. && \
+    rm -rf libtinfo
+
+# Build libunwind from source. The packaged version breaks linking
```

Comment:
can you link to a ticket in llvm or similar so that we know how this breaks in the future?

Reply:
It looks like this has been fixed since I first made the change, removed!